TemplateApplication/Contacts/Contacts.swift

```diff
@@ -10,8 +10,12 @@ import Foundation
 import SpeziContact
 import SwiftUI
 
+/// Display contacts for your app.
+///
+/// Modify the contact details for your app in the Contacts view.
+///
+/// ![A screenshot of the Contact screen](Contacts)
 
```

Comment:
To render the documentation it would require to remove the empty line between the docs & the type. Also applies to other files.